backend/compose.yaml

```diff
@@ -4,4 +4,24 @@ services:
       context: .
       dockerfile: ./Containerfile
     ports:
-      - 8080:8080 # API
+      - 8080:8080
+    depends_on:
+      - db
+    environment:
+      - DB_HOST=db
+      - DB_PORT=5432
+      - DB_USER=testuser
+      - DB_PASSWORD=testpassword
+      - DB_NAME=testdb
+
+  db:
+    image: postgres:15
+    environment:
+      POSTGRES_USER: testuser
+      POSTGRES_PASSWORD: testpassword
+      POSTGRES_DB: testdb
+    ports:
+      - 5432:5432
+    volumes:
+      - ./db_data:/var/lib/postgresql/data
+      - ./init_db:/docker-entrypoint-initdb.d/
```

Comment:
I have a feeling we are missing an ENTRYPOINT here. I tried to turn on the mock database and connect to it using pgAdmin. It did connect (and I presume I connected to the right one because it needs password). But there are no SQL data
![Untitled](https://github.com/user-attachments/assets/8b55d120-0de8-4601-8109-de1d03252f60)